cypress/support/fragments/data_import/match_profiles/newMatchProfile.js

```diff
@@ -62,10 +62,8 @@ const fillMatchProfileForm = ({
     fillExistingRecordFields(existingRecordFields.in2, 'in2');
     fillExistingRecordFields(existingRecordFields.subfield, 'subfield');
```

Comment:
please fix linter issue:
'getLongDelay' is defined but never used. Allowed unused vars must match /React/u

Reply:
Done